Inferelator/inferelator.R

```diff
@@ -0,0 +1,486 @@
+## Bonneau lab 
+## NYU - Center for Genomics and Systems Biology
+
+# Call this script with a job config file as arguments
+# Example call: Rscript inferelator.R jobs/dream4_cfg.R
+
+
+library('Matrix')
+
+rm(list=ls())
+gc()
+
+source('R_scripts/utils.R')
+source('R_scripts/design_and_response.R')
+source('R_scripts/priors.R')
+source('R_scripts/mi_and_clr.R')
+source('R_scripts/bayesianRegression.R')
+source('R_scripts/men.R')
+source('R_scripts/evaluate.R')
+source('R_scripts/tfa.R')
+source('R_scripts/group_predictors.R')
+source('R_scripts/summarize_results.R')
+#source('R_scripts/vis_tfs_and_targets.R')
+
+
+date.time.str <- format(Sys.time(), "%Y-%m-%d_%H-%M-%S")
+print(date.time.str)
+start.proc.time <- proc.time()
+
+# default job parameters
+PARS <- list()
+
+PARS$input.dir <- 'input/dream4'
+
+PARS$exp.mat.file <- 'expression.tsv'
+PARS$tf.names.file <- 'tf_names.tsv'
+PARS$meta.data.file <- NULL
+PARS$priors.file <- NULL
+PARS$gold.standard.file <- NULL
+PARS$leave.out.file <- NULL
+PARS$randomize.expression <- FALSE
+
+PARS$job.seed <- 42  # set to NULL if a random seed should be used
+PARS$save.to.dir <- NULL
+PARS$num.boots <- 20
+PARS$max.preds <- 10
+PARS$mi.bins <- 10
+PARS$cores <- 8
+
+PARS$delT.max <- 110
+PARS$delT.min <- 0
+PARS$tau <- 45
+
+PARS$perc.tp <- 0
+PARS$perm.tp <- 1
+PARS$perc.fp <- 0
+PARS$perm.fp <- 1
+PARS$pr.sel.mode <- 'random'  # prior selection mode: 'random' or 'tf'
+
+PARS$eval.on.subset <- FALSE
+
+PARS$method <- 'BBSR'  # 'BBSR' or 'MEN'
+PARS$prior.weight <- 1
+
+PARS$use.tfa <- FALSE
+PARS$prior.ss <- FALSE
+
+PARS$output.summary <- FALSE
```

Comment:
Hey Aaron, can we set this to true by default? This generates a condensed file of the network, with only the highest confidence edges. It shouldn't rely on any of the html, markdown, or knittr packages